### ISSUE TITLE
Problems with naf agent when upgrading tensorflow to version 1.11.0

### ISSUE THREAD

**Issue by gvgramazio** [keras-rl/keras-rl#260]
Upgrading from tensorflow 1.10.1 to 1.11.0 I have the following error running example `naf_pendulum.py` with python3.

```
[...]
Traceback (most recent call last):
  File "naf.py", line 106, in <module>
    agent.compile(Adam(lr=.001, clipnorm=1.), metrics=['mae'])
  File "/keras-rl/rl/agents/dqn.py", line 615, in compile
    A_out = NAFLayer(self.nb_actions, mode=self.covariance_mode)([L_out, mu_out, a_in])
  File "/usr/local/lib/python3.6/dist-packages/keras/engine/topology.py", line 619, in __call__
    output = self.call(inputs, **kwargs)
  File "/keras-rl/rl/agents/dqn.py", line 433, in call
    L_flat = tf.concat(1, [zeros, L_flat])
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/ops/array_ops.py", line 1122, in concat
    tensor_shape.scalar())
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/tensor_shape.py", line 848, in assert_is_compatible_with
    raise ValueError("Shapes %s and %s are incompatible" % (self, other))
ValueError: Shapes (2, ?, 1) and () are incompatible
```

No problems at all with tensorflow version 1.10.1. I'm up to date with keras-rl (master branch)  and keras (version 2.2.4).

**Comment by mirraaj** [comment]
The issue is arising from Keras backend. Have you updated keras?

**Comment by gvgramazio** [comment]
I installed keras through pip, so I've not built it from source. Right now, the newest version available through pip in my system (Ubuntu 18.04) is 2.2.4 that is what I'm currently using.

**Comment by mirraaj** [comment]
@gvgramazio, This issue has been solved. Please update with the master branch. Thanks. Let me know if any issues occur.
